templates/appliance/shared/_steps_nuc_flash_usb.html

```diff
@@ -5,8 +5,8 @@ <h4 class="p-stepped-list__title">
   </h4>
   <div class="p-stepped-list__content">
     <ol>
-      <li>Download and copy the <a href="/download/desktop">Ubuntu Desktop 20.04 LTS image</a> on the first USB flash drive by following the live USB Ubuntu Desktop tutorial for {% if os=='ubuntu' %}<a href="/tutorials/tutorial-create-a-usb-stick-on-ubuntu">Ubuntu</a>{% elif os=='windows' %}<a href="/tutorials/tutorial-create-a-usb-stick-on-windows">Windows</a>{% else %}<a href="/tutorials/tutorial-create-a-usb-stick-on-macos">macOS</a>{% endif %}</li>
-      <li>Copy the Ubuntu Core image for Intel NUC on the second USB flash drive</li>
+      <li>Download and copy the <a href="/download/desktop">Ubuntu Desktop 20.04 LTS image</a> on the first USB flash drive by following the live USB Ubuntu Desktop {% if os=='ubuntu' %}<a href="/tutorials/tutorial-create-a-usb-stick-on-ubuntu">tutorial for Ubuntu</a>{% elif os=='windows' %}<a href="/tutorials/tutorial-create-a-usb-stick-on-windows">tutorial for Windows</a>{% else %}<a href="/tutorials/tutorial-create-a-usb-stick-on-macos">tutorial for macOS</a>{% endif %}</li>
+      <li>Copy the Ubuntu Appliance image you downloaded to get here on to the second USB flash drive</li>
```

Comment:
The sentence doesn't make sense. I think its the "to get here on to" part.

Reply:
good catch, I just removed the 'to get here'